docusaurus.config.js

```diff
@@ -62,15 +62,15 @@ const config = {
             items: [
               {
                 label: "Twitter",
-                href: "https://twitter.com/SablierHQ",
+                href: "https://twitter.com/sablierhq",
               },
               {
                 label: "Discord",
                 href: "https://discord.gg/bSwRCwWRsT",
               },
               {
                 label: "Blog",
-                href: "https://blog.sablier.finance",
```

Comment:
It was actually my fault here. I set the URL to `blog.sablier.finance`. Sorry!

Reply:
Which is an excellent opportunity to create this subdomain on Medium! Would already make us less dependent on Medium and benefit our SEO more.